articles/openshift/howto-using-aro.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Using `az aro` | Microsoft Docs
+description: Create, access, and delete clusters with `az aro` extension
+author: klamenzo 
+ms.author: b-lejaku
+ms.service: container-service
+ms.topic: conceptual
+ms.date: 03/02/2020
+keywords: aro, openshift, az aro, red hat, cli
+#Customer intent: As a customer, I want to create an ARO custer using the command line.
+---
+
+# Using `az aro`
+
+The `az aro` extension allows you to create, access, and delete Azure Red Hat OpenShift clusters directly from the command line using the Azure CLI.
+
+## Installing the extension
+
+1. Install the [`az`](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) client, if you have not already. You will need `az` version 2.0.72 or greater, as this version includes the `az network vnet subnet update --disable-private-link-service-network-policies` flag.
+
+2. Log in to Azure.
+
+```
+az login
+```
+
+3. Run the following command to install the `az aro` extension:
+
+```
+az extension add --source https://arosvc.blob.core.windows.net/az-preview/aro-0.1.0-py2.py3-none-any.whl
+```
+
+4. Verify the ARO extension is registered.
+
+```
+az -v
+...
+Extensions:
+aro                                0.1.0 (dev) /path/to/rp/python/az/aro
+...
+Development extension sources:
+    /path/to/rp/python
+...
+```
+
+
+## Registering the resource provider
+
+To opt-in for the preview you need to register the `Microsoft.RedHatOpenShift` resource provider.
+
+```
+az provider register -n Microsoft.RedHatOpenShift --wait
+```
+
+
+## Prerequisites to create an Azure Red Hat OpenShift v4 cluster
+
+You need the following items to create an Azure Red Hat OpenShift v4 cluster.
+
+* A cluster AAD application (client ID and secret) and service principal, or sufficient AAD permissions for `az aro` create to create these for you automatically.
```

Comment:
typo: `create to create`

Reply:
Trying to read this through the eyes of someone new to the service.  I don't think most people will understand this.

A diagram would be super useful here, we could sketch one out together?

Also, maybe refactoring to a couple of different paths:
1) happy newbie: I either have a pre-existing vnet, or I don't, but either way it doesn't have the cluster subnets.  I'm not in a locked down corporate azure environment, I own my subscription, I don't have to understand about resource provider service principals, let's go!

2) (separate PR?) I'm in a locked down environment and have to worry about all the gruesome detail.